sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -23,6 +23,16 @@
      */
     public static final String PROPERTY_HTTPS_PROXY = "HTTPS_PROXY";
 
+    /**
+     * Endpoint to connect to when using Azure Active Directory managed service identity (MSI).
+     */
+    public static final String PROPERTY_IDENTITY_ENDPOINT = "IDENTITY_ENDPOINT";
```

Comment:
If this is specific to AAD, can this constant be declared in azure-identity where we have everything related to AAD?